ratis-proto/src/main/proto/Raft.proto

```diff
@@ -175,6 +175,13 @@ message CommitInfoProto {
   uint64 commitIndex = 2;
 }
 
+message PeerInfoProto {
+  CommitInfoProto commitInfo = 1;
+  uint64 appliedIndex = 2;
+  uint64 currentTerm = 3;
+  uint64 snapshotIndex = 4;
+}
```

Comment:
Since the word `PeerInfo` is already used, let's call it `LogInfoProto`.  BTW, let's use `TermIndexProto` instead of `uint64`.  Then, everything will have a term.
```
message LogInfoProto {
  TermIndexProto lastSnapshot = 1;
  TermIndexProto applied = 2;
  TermIndexProto committed = 3;
  TermIndexProto lastEntry = 4;
}
```

Reply:
sure, updated!